tests/functional/test.cli.run.js

```diff
@@ -53,8 +53,9 @@ describe('web-ext run', () => {
     const argv = [
       'run',
       '--target', 'firefox-desktop',
-      '--target', 'not-supported',
```

Comment:
(minor)
Why did this `--target not-supported` flag move? I asked for it to be put in between valid arguments (in the original PR), because an invalid argument amidst valid arguments covers more scenarios than an invalid parameter in the end.

Reply:
I moved it because if the invalid parameter is in the middle then all the targets that follows are never reached.

The test would clearly pass without this change... but there will be no test which ensures that `firefox-android` (and now `chromium`) are considered valid targets.